ui/src/utils/cluster.utils.ts

```diff
@@ -21,7 +21,6 @@ export function generateClusterName(username = ''): string {
 
   // ROX-15492
   //   - OCP 3.11 flavor allows us a maximum of up to 28 characters to fulfil: "openshift_public_hostname must be 63 characters or less".
-  //   - OCP 4 demo: Let's Encrypt allows common names of up to 63 characters. With the format '*.apps.<name>.openshift.infra.rox.systems', 28 characters remain for the name.
```

Comment:
It is incorrect though because we don't Lets Encrypt the OCP 4 demos. I'm not sure if that was ever done. Maybe manually by some folks. And now we have Stephans ACME certs.